contrib/docker/README.md

```diff
@@ -0,0 +1,39 @@
+# Getting Start with Superset using Docker
+
+Docker is the recommended way to get stated with Superset.
```

Comment:
This is contradictory with the note in `installation.rst`, so far the docker in `contrib/` has not been super reliable and supported by the community. Let's have sustained support on this for a while before making it the recommended way.

Reply:
I took this in `installatio.rst` as an endoresement :) I'll change it.

```
Start with Docker
-----------------
```